test/fixtures/served/iframe.html

```diff
@@ -52,6 +52,12 @@
             sentinel: sentinel,
             type: 'send-embed-state',
           }, '*');
+    } else if (event.data.type === 'requestPymjsHeight') {
+      var msg = [ 'pym', 'height', event.data.height ].join( 'xPYMx' );
```

Comment:
You need to add some bogus `id` after `pym` and before `height`.

Reply:
Fixed in 2d578da50ec1cb3cb7626d98683aea9fba030568